src/app/games/lodestone/lodestone.component.html

```diff
@@ -48,58 +52,60 @@
-                         [ngClass]="lodestone.movingClass"
-                         [attr.points]="TRIANGLE_IN"
+                         [attr.points]="lodestone.direction === 'push' ? TRIANGLE_OUT : TRIANGLE_IN"
                          attr.transform="rotate({{ i * 90 }})" />
             </g>
         </g>
     </g>
     <g *ngFor="let plate of viewInfo.pressurePlates"
-       id="pressurePlate_{{ plate.position }}">
-        <g *ngFor="let square of plate.coords; index as i">
-            <rect id="plateSquare_{{ plate.position }}_{{ i }}"
-                  class="base"
-                  (click)="(square.temporary) ? deselectPressurePlate(plate.position, i) : selectPressurePlate(plate.position, i)"
-                  [ngClass]="square.squareClasses"
-                  [attr.x]="(SPACE_SIZE * square.coord.x) + STROKE_WIDTH"
-                  [attr.y]="(SPACE_SIZE * square.coord.y) + STROKE_WIDTH"
-                  [attr.width]="SPACE_SIZE"
-                  [attr.height]="SPACE_SIZE"
-                  rx="20"
-                  ry="20" />
-            <circle *ngIf="square.hasPiece"
-                    id="platePiece_{{ plate.position }}_{{ i }}"
-                    (click)="(square.temporary) ? deselectPressurePlate(plate.position, i) : selectPressurePlate(plate.position, i)"
-                    class="base"
-                    [ngClass]="square.pieceClasses"
-                    [attr.r]="PIECE_RADIUS"
-                    [attr.cx]="(SPACE_SIZE * square.coord.x) + STROKE_WIDTH + (SPACE_SIZE / 2)"
-                    [attr.cy]="(SPACE_SIZE * square.coord.y) + STROKE_WIDTH + (SPACE_SIZE / 2)" />
+       id="pressurePlate_{{ plate.position }}" >
+        <g *ngFor="let square of plate.coords; index as i"
+           attr.transform="translate({{ viewInfo.pressurePlateShift[plate.position].x * SPACE_SIZE }}, {{ viewInfo.pressurePlateShift[plate.position].y * SPACE_SIZE }})" >
+            <g id="plate_{{ plate.position }}_{{ i }}"
+               (click)="(square.temporary) ? deselectPressurePlate(plate.position, i) : selectPressurePlate(plate.position, i)" >
+                <rect id="plateSquare_{{ plate.position }}_{{ i }}"
+                      class="base"
+                      [ngClass]="square.squareClasses"
+                      [attr.x]="(SPACE_SIZE * square.coord.x) + STROKE_WIDTH"
+                      [attr.y]="(SPACE_SIZE * square.coord.y) + STROKE_WIDTH"
+                      [attr.width]="SPACE_SIZE"
+                      [attr.height]="SPACE_SIZE"
+                      rx="20"
+                      ry="20" />
+                <circle *ngIf="square.hasPiece"
+                        id="platePiece_{{ plate.position }}_{{ i }}"
+                        class="base"
+                        [ngClass]="square.pieceClasses"
+                        [attr.r]="PIECE_RADIUS"
+                        [attr.cx]="(SPACE_SIZE * square.coord.x) + STROKE_WIDTH + (SPACE_SIZE / 2)"
```

Comment:
maintenant qu'on sais faire des composant enfants en svg (bordel, seulement maintenant haha, après 2400h de dev collectif :D ) on pourra éventuellement faire des truc genre, mettre en commun les logique des cases avec ou sans stroke, ou le stroke externe du cercle rencontre le stroke interne de la case, tout ça, éventuellement.

Reply:
oui ça sera cool! Ça sera pour la v2 du ticket "games to enhance"? :D